keras_contrib/applications/resnet.py

```diff
@@ -12,11 +12,16 @@
 - [Deep Residual Learning for Instrument Segmentation in Robotic Surgery](https://arxiv.org/abs/1703.08580)
   for segmentation support.
 
-Implementation Adapted from: github.com/raghakot/keras-resnet
+Implementation adapted from: github.com/raghakot/keras-resnet
+
+TimeDistributed adapted from: github.com/voletiv/keras-resnet
 """
 from __future__ import division
 
 import six
+import warnings
+from keras.utils.data_utils import get_file
+from keras.utils import layer_utils
```

Comment:
22-24 (import warnings, get_file, layer_utils) are unnecessary I think, because they are not used anywhere